definition_validator/validator.cc

```diff
@@ -905,34 +905,30 @@ class ValidateWalk {
         }
     }
 
-    core::AutocorrectSuggestion suggestMethodDefinition(const core::GlobalState &gs,
-                                                        const core::Loc classOrModuleDefLoc,
-                                                        const core::MethodRef abstractMethodRef) {
-        auto [_, indentLength] = classOrModuleDefLoc.findStartOfLine(gs);
-        string indent(indentLength, ' ');
+    core::AutocorrectSuggestion::Edit defineInheritedAbstractMethod(const core::GlobalState &gs,
+                                                                    const core::MethodRef abstractMethodRef,
+                                                                    const core::Loc insertAt, const string format,
+                                                                    const string classOrModuleIndent) {
```

Comment:
Is it intentional that we're taking these two `string` arguments by copy?

To avoid copying them, you can either take them by `const string &` or by `const string_view` (which `string` can automatically convert to).

Reply:
Not intentional unfortunately: I'm still learning to be sensitive to these sorts of concerns. This type of scrutiny is very appreciated! Fixed in 0e32270.